### PR TITLE
Added tests for utils.py

### LINKED ISSUE #11: Add some unit tests for functions in crash_similarity.py

### PR DESCRIPTION
**Pull request by mansimarkaur**
I'm facing an issue in importing the modules from the root directory. flake8 is giving an error related to this: `module level import not at top of file`.

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import sys
+sys.path.append('../')
```
Comment (marco-c) [inline tests/test_utils.py]: Remove these two lines, they are not needed if you run the tests using `python2 -m unittest tests.test_utils` or `python3 -m unittest tests/test_utils.py`.

File: tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        bug_id = '1308863'
+        resp = utils.get_with_retries(self.url, params={'id': bug_id})
+        self.assertEqual(resp.status_code, 200)
+
+    def test_get_with_retries_raises_400_with_no_params(self):
+        resp = utils.get_with_retries(self.url)
+        self.assertEqual(resp.status_code, 400)
+
+    def test_read_files(self):
+        paths = ['crashsimilarity_data/firefox-crashes-2016-11-09.json.gz']
```
Comment (marco-c) [inline tests/test_utils.py]: Can you create a file specifically for tests? Like `tests/test.json` from https://github.com/marco-c/crashsimilarity/pull/50/files.

Comment (mansimarkaur) [inline tests/test_utils.py]: I've added a `test_utils.json` file specifically for the test.

Comment (marco-c) [inline tests/test_utils.py]: Thanks! You forgot to `git add` it though 😄

File: .travis.yml [hunk 3]
```diff
@@ -7,4 +7,6 @@ install:
     - pip install --only-binary=numpy,scipy
     - pip install -r requirements.txt
     - pip install -r test-requirements.txt
-script: flake8
+script:
+    - python -m unittest discover tests
```
Comment (marco-c) [inline .travis.yml]: Thanks, this will fix #37.

File: tests/test_utils.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        bug_id = '1308863'
+        resp = utils.get_with_retries(self.url, params={'id': bug_id})
+        self.assertEqual(resp.status_code, 200)
+
+    def test_get_with_retries_raises_400_with_no_params(self):
+        resp = utils.get_with_retries(self.url)
+        self.assertEqual(resp.status_code, 400)
+
+    def test_read_files(self):
+        paths = ['tests/test_utils.json']
```
Comment (marco-c) [inline tests/test_utils.py]: Don't forget to commit this file too

Comment (mansimarkaur) [inline tests/test_utils.py]: I'm really sorry. The file was being ignored by `.gitignore`. I've pushed it now.

Comment (marco-c) [inline tests/test_utils.py]: No problem 😃

**Pull request merged**